Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnection.cpp

```diff
@@ -34,9 +36,27 @@ namespace Multiplayer
         m_networkEditorInterface = AZ::Interface<INetworking>::Get()->CreateNetworkInterface(
             AZ::Name(MpEditorInterfaceName), ProtocolType::Tcp, TrustZone::ExternalClientToServer, *this);
         m_networkEditorInterface->SetTimeoutMs(AZ::TimeMs{ 0 }); // Disable timeouts on this network interface
-        ActivateDedicatedEditorServer();
-    }
 
+        // Wait to activate the editor-server until LegacySystemInterfaceCreated so that the logging system is ready
+        // Automated testing listens for these logs
+        if (editorsv_isDedicated)
+        {
+            // If the settings registry is not available at this point,
+            // then something catastrophic has happened in the application startup.
```

Comment:
I regret writing this comment now. It's going to show up everywhere as people wire in to the application lifecycle events.

No change necessary, I'm just taking a note for myself in the future to expect this to happen more often.